rag/advanced_rag/harness/orchestrator/decompose.py

```
@@ -216,25 +254,32 @@ async def decompose_and_search(state: dict, tools) -> dict:
+            # the exact evidence each claim referenced, not a global prefix.
+            cited_evidence_ids: list[str] = []
+            for r in agent_results:
+                cited_evidence_ids.extend(r.evidence_ids or [])
+            try:
+                grounded = await llm_grounded_verify(
+                    tools,
+                    ctx.question,
+                    [(r.claim_id, r.report or "") for r in agent_results if r.report],
+                    cited_evidence_ids or None,
+                )
+            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)